src/citrine/__init__.py

```diff
@@ -1,5 +1,6 @@
 # TODO: Add a docstring here
 from citrine.citrine import Citrine  # noqa: F401
 import logging
+from .__version__ import __version__
```

Comment:
Build failures are coming from this import being unused

Reply:
👍